spec/index.md

```diff
@@ -305,13 +278,19 @@ Each dimension is represented by a hash in the `dimensions` array. The hash has
     "primaryKey": ["Project", "ClassCode"],
 
     // OPTIONAL: Describes what kind of a dimension it is. `dimensionType` is a string that `MUST` be one of the following:
-    // datetime, entity, classification, activity, fact, location, other`
-    "dimensionType": "classification",
+    // * "datetime": the date of a transaction 
+    // * "entity": names the organisation doing the spending or receiving
+    // * "classification": one or more fields that create a categorical hierarchy of the type of spending (eg, Health > Hospital services > Nursing) 
+    // * "activity": one or more fields that create an administrative hierarchy (eg, Department > Programme > Project)
```

Comment:
For "classification" dimensions, there is an associated "classificationType" of which "administrative" is one of the possible values.  This should probably be made clearer in the spec.  The classificationType stuff is currently in the "content" section and I think it should brought up here to flesh out all the possible ways to describe a dimension.  What do you think @rgrp?